### ISSUE TITLE
Nice, but syntax not like the official Primer site

### ISSUE THREAD

**Issue by lombardi-gux** [karelvuong/st-primer#15]
The theme is very well done, but the syntax theme (while nice) is not the Primer color scheme as displayed on the official site: http://primercss.io/

For example, here's a snippet in your theme
![screen shot 2015-03-30 at 9 07 54 am](https://cloud.githubusercontent.com/assets/7459986/6896739/8cdb0b2c-d6bc-11e4-8ca6-b25747c3943f.png)

And the same snippet taken from the Primer site [here](http://primercss.io/buttons/)
![screen shot 2015-03-30 at 9 08 03 am](https://cloud.githubusercontent.com/assets/7459986/6896759/a87e323c-d6bc-11e4-97c8-7e428be2c213.png)

**Comment by ghost** [comment]
Thanks for this, @lombardi-gux. 

I actually based the syntax theme off of GitHub's syntax highlighting used in issues and pull requests as there were more languages to go off of.

Where Primer does come in to play is in the color scheme used in accents. This can be seen on http://primercss.io/colors.

What do you think of a secondary light them that makes use of these colors as seen in your second screenshot?

**Comment by lombardi-gux** [comment]
Makes sense.  That might be a good idea.

I had to modify the colors a bit for html, using a combo of your colors, a primer color, and the green for "tags" sampled off a live repo.  The html colors in the Primer theme are a little hard to work with.  Here's mine:

![screen shot 2015-03-31 at 8 59 03 am](https://cloud.githubusercontent.com/assets/7459986/6919351/38ee085c-d784-11e4-8696-bd59eb9b9596.png)

Also, there are 2 annoyances in every theme which I fixed here: http://cl.ly/2r0x3j1u462q/primer.light2.tmTheme
1. double ampersands in strings for logical comparisons are flagged as invalid
2. escaped strings in CSS  (`\0020`)

Not your error, all themes seem to miss that, perhaps the tmLanguages are coded wrong.

Thanks.
